### ISSUE TITLE
BGA

### ISSUE THREAD

**Issue by Hendirk** [Stephan-S/FS22_AutoDrive#479]
Leider werden immer noch nicht bei der BGA die Ballen abgeladen 
![Ballen BGA](https://user-images.githubusercontent.com/47453761/162582024-d53c1e85-9319-4954-bb2d-10f932557667.png)

**Comment by NeonRider1999** [comment]
Probiere  mal den Zielpunkt weiter nach hinten zu schieben vlt. hilft es

**Comment by Hendirk** [comment]
Wie mache ich das, ich finde die Funktion nicht mehr


Gesendet von Mail für Windows

Von: NeonRider1999
Gesendet: Samstag, 9. April 2022 18:27
An: Stephan-S/FS22_AutoDrive
Cc: Hendrik; Author
Betreff: Re: [Stephan-S/FS22_AutoDrive] BGA (Issue #479)

Probiere mal den Zielpunkt weiter nach hinten zu schieben vlt. hilft es

**Comment by NeonRider1999** [comment]
Den Editor Modus öffnen (Symbol unten Rechts in der Ecke vom AD Fenster) dann mit der Maus auf den Punkt gehen und mit der Rechten Maus Taste den Punkt Verschieben

**Comment by Axel32019** [comment]
@Hendirk Funktioniert es?

**Comment by Hendirk** [comment]
Ja, hab es nach mehreren Versuchen irgendwie hin bekommen Danke für die Hilfe --Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 25.04.22, 20:14 schrieb Axel32019 ***@***.***>:

   
  @Hendirk Funktioniert es?
